### ISSUE TITLE
Upgrade vega-tooltip to 0.30.0 to support custom tooltips

### ISSUE THREAD

**Issue by jovancacvetkovic** [opensearch-project/OpenSearch-Dashboards#3358]
**Is your feature request related to a problem? Please describe.**
The current vega-tooltip version 0.24.0 doesn't support custom tooltip rendering.

**Describe the solution you'd like**
Upgrade the vega-tooltip package to version 0.30.0.

**Additional context**
Also related to [#3317](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/3317)

**Comment by joshuarrrr** [comment]
@jovancacvetkovic Thanks for suggesting and opening the PR. I agree that getting updated to the latest version will help us have access to the latest features.

Can you summarize and link to any potential breaking changes between `0.24.0` and `0.30.0` that we should consider? Because other OpenSearch dashboard plugins may also be directly depending on and using the current version.

**Comment by jovancacvetkovic** [comment]
> 

@joshuarrrr Hey, thanks for reminding me to add more context to this, should have done that already.
Here are the latest changes for the 0.30.0 version [v0.30.0 changes](https://github.com/vega/vega-tooltip/releases/tag/v0.30.0)
And the code [changes](https://github.com/vega/vega-tooltip/compare/v0.24.0...v0.30.0)

**To summarize**
Most changes are about updating versions for dependencies to follow the vega-lite dependencies and bug fixes, major ones are: 
- make sure there is only one instance of the tooltip [#626]( https://github.com/vega/vega-tooltip/pull/626)
- and changing the inline style so that these are not affected by CSP [#714](https://github.com/vega/vega-tooltip/pull/714)
Two major updates are: 
- **moving from node-sass to sass** [#614](https://github.com/vega/vega-tooltip/pull/614)
- adding a new Handler fn called **formatTooltip**, which defaults to the **formatValue** _which has not been changed_ from version 0.24.0. Meaning, if you don't add the new fn (to customize the tooltip) it will use the old formatting fn. [#557](https://github.com/vega/vega-tooltip/pull/557)

**Comment by joshuarrrr** [comment]
@jovancacvetkovic Thanks for the summary - seeing the changes in that form makes me even more eager to make this update 😁
